### DISCUSSION TITLE
Can you please do a patch for ledger for the 0.x version of wagmi?

### DISCUSSION THREAD

**Discussion by fionnachan** [wevm/wagmi#3311] [Ideas]
We aren't on v1 yet but would really love to get a patch
context: https://x.com/bantg/status/1735279127752540465?s=20

**Comment by fionnachan** [comment]
@tmm and @jxom

**Reply by tmm** [reply]
If you don't use the `LedgerConnector`, then there should be no issue. The Ledger team has fixed the issue upstream as well.

**Comment by bamnenim-kryptonium** [comment]
https://github.com/wevm/wagmi/releases/tag/wagmi%401.4.12

The team hot-fixed this issue here

**Reply by hkrawczyk** [reply]
removing something that was fixed in 6h isn't hot-fixing. 
Now we'll wait another 72h to XXXh till someone will enable it back.

**Reply by bamnenim-kryptonium** [reply]
The team removed the ledger connect tkit loader just in 30 min from the first notice of the issue. - so it was 5 hours before the issue was resolved.
I'm not sure what else do u call hotfix if it was not a hotfix. :P

**Reply by hkrawczyk** [reply]
@bamnenim-kryptonium what I'm saying is that after 4 days it is still disabled...
so the hot-fix is breaking the long-term functionality of the product.

**Reply by tmm** [reply]
You are welcome to fork the [`LedgerConnector`](https://github.com/wevm/wagmi/blob/e69f317971c57eb9002e866f01d3c23ca2b0c748/packages/connectors/src/ledger.ts) into your codebase. We likely won't add it back to Wagmi until we are satisfied with security practices from the Ledger team. In the meantime, you can still connect to Ledger through the `WallectConnectConnector`.

**Reply by Wozacosta** [reply]
@tmm Hey! 

>  We likely won't add it back to Wagmi until we are satisfied with security practices from the Ledger team 

Do you expect more than what was is listed in the "Remedial actions" part of the December's exploit postmortem ([link here](https://www.ledger.com/blog/security-incident-report))? I understand internal audits are hard to track externally and the only real measure will be the time without an exploit, but some concrete measures have already been implemented: 

The ledger connect-kit repo now monitors if the published version is different from the github repo one. [Here's the github action](https://github.com/LedgerHQ/connect-kit/actions/workflows/check_npm_repo.yml).

Would you be open to a PR introducing the LedgerConnector (updated for the v2) that loads the @ledgerhq/connect-kit package (pinning its version in a package.json dependencies).

**Reply by ComradeAERGO** [reply]
> We likely won't add it back to Wagmi until we are satisfied with security practices from the Ledger team

Hi there @tmm, I work at Ledger with @Wozacosta and we're currently working on a PR to bring the connector back to wagmi v2.

Can you please detail the "security practices" you require to make our PR acceptable for you?

**Reply by tmm** [reply]
You tell us! Feel free to detail it in your PR and we can go from there. Want to see if our thinking aligns before sharing.
